buildspec/buildspec.yml

```diff
@@ -0,0 +1,42 @@
+version: 0.2
+
+env:
+  secrets-manager:
+    SONAR_TOKEN: Codebuild-DSS:SONAR_TOKEN
+    DSSMAINS3URI: Codebuild-DSS:DSSMAINS3URI
+  variables:
+    DSSGLOBLIST: "nkv-datamover-*.tgz dss_client-*.tgz"
+
+phases:
+  build:
+    commands:
+      - |
+        build-wrapper-linux-x86-64 --out-dir bw-output \
+          ./dss_client/scripts/build.sh
+      - |
+        sonar-scanner \
+          -Dsonar.organization=openmpdk \
+          -Dsonar.projectKey=OpenMPDK_dss-ecosystem \
+          -Dsonar.sources=. \
+          -Dsonar.cfamily.build-wrapper-output=bw-output \
+          -Dsonar.host.url=https://sonarcloud.io \
+          -Dsonar.pullrequest.github.summary_comment=true \
+          -Dsonar.pullrequest.provider=GitHub \
+          -Dsonar.pullrequest.key=$(echo $GITHUB_REF | grep -oP "^refs/pull/\K[^/]+") \
+          -Dsonar.pullrequest.base=${GITHUB_BASE_REF} \
+          -Dsonar.pullrequest.branch=${GITHUB_HEAD_REF} \
+          -Dsonar.pullrequest.github.repository=${GITHUB_REPOSITORY}
+
+      # Create dss-datamover Artifact
```

Comment:
Please create a new buildspec to create the dss-datamover Artifact. It should be called "datamover.yml"
Please update the DSSGLOBLIST var for each buildspec so that it appropriately references the artifacts from each buildspec, as well as the artifacts section below.

the `statemainartifacts.sh` script should still be run in both buildspec files.

Reply:
Obviously this will require updating the Github workflows to reference the additional buildspecs.